_inc/client/main.jsx

```diff
@@ -330,17 +338,16 @@ export default connect(
 window.wpNavMenuClassChange = function() {
 	let hash = window.location.hash;
 	const settingRoutes = [
-			'#/settings',
-			'#/general',
-			'#/discussion',
-			'#/security',
-			'#/performance',
-			'#/traffic',
-			'#/writing',
-			'#/sharing',
-			'#/privacy',
-		],
-		dashboardRoutes = [ '#/', '#/dashboard', '#/my-plan', '#/plans' ];
+		'#/settings',
+		'#/general',
```

Comment:
I take it some of these routes are old?

Reply:
Yeah these green lines are something are just modified due to `prettier` updating them. Will fix the old ones